core/src/main/java/org/apache/iceberg/MetadataColumns.java

```diff
@@ -44,6 +44,7 @@ private MetadataColumns() {
       Integer.MAX_VALUE - 101, "file_path", Types.StringType.get(), "Path of a file in which a deleted row is stored");
   public static final NestedField DELETE_FILE_POS = NestedField.required(
       Integer.MAX_VALUE - 102, "pos", Types.LongType.get(), "Ordinal position of a deleted row in the data file");
+  public static final String DELETE_FILE_ROW_FIELD_NAME = "row";
```

Comment:
@rdblue, did we not add the name on purpose?

Reply:
I don't think so

Reply:
Ok, keeping it then.